R/prep_data.R

```diff
@@ -74,6 +75,10 @@
 #'   outcome are 0 or 1 they will be converted to factor with levels N and Y for
 #'   classification. Note that which level is the positive class is set in
 #'   training functions rather than here.
+#' @param ref_levels A named vector. Name is the feature and the desired refence
```

Comment:
I think this can be clearer. Try starting with this: "A named vector that provides the reference levels for categorical variables. If `make_dummies` is TRUE, the reference level will not have a column created, making model estimates for other levels relative to the reference level. By default, ..."

Reply:
Thank you. I like that a lot better.